protocols/directdefund/directdefund.go

```diff
@@ -159,8 +159,8 @@ func (o Objective) Reject() protocols.Objective {
 	return &updated
 }
 
-// OwnsChannel returns the channel that the objective is funding.
-func (ddo Objective) OwnsChannel() types.Destination {
+// PrincipalChannel returns the channel that the objective is funding.
+func (ddo Objective) PrincipalChannel() types.Destination {
```

Comment:
We probably want all of these godoc lines to match, and to do a better job at explaining what this is. I think we probably want an ADR, too.